sdcm/utils/version_utils.py

```diff
@@ -19,6 +20,7 @@
 REPO_VERSIONS_REGEX = re.compile(r'Version: (.*?)\n', re.DOTALL)
 
 SCYLLA_VERSION_RE = re.compile(r"\d+(\.\d+)?\.[\d\w]+([.~][\d\w]+)?")
+SCYLLA_SSTABLE_SUPPORTED_RE = re.compile(r'Feature (.*)_SSTABLE_FORMAT is enabled')
```

Comment:
`SCYLLA_SSTABLE_SUPPORTED_RE` -> `SSTABLE_FORMAT_VERSION_REGEX`

Reply:
Bentsi's suggestion is better, the matched result might be 'LA', 'MC'.

Reply:
fixed